core/domain/services/graphql/types/Datetime.php

```diff
@@ -49,6 +49,12 @@ public function getFields()
         return [
             new GraphQLField(
                 'id',
+                ['non_null' => 'ID'],
+                null,
+                esc_html__('The globally unique ID for the object.', 'event_espresso')
+            ),
+            new GraphQLOutputField(
+                lcfirst($this->name()) . 'Id',
```

Comment:
why not just hardcode `datetimeId` ?

Reply:
As mentioned in [this card](https://github.com/eventespresso/event-espresso-core/projects/6#card-29371659), we can change the type name. So, I set it dynamically so as not to miss those when changing the type names.